admittance_controller/CMakeLists.txt

```
@@ -23,6 +23,7 @@ set(THIS_PACKAGE_INCLUDE_DEPENDS
   tf2_geometry_msgs
   tf2_kdl
   tf2_ros
+  tinyxml2
```

```
   --- stderr: admittance_controller
  CMake Error at CMakeLists.txt:33 (find_package):
    By not providing "Findtinyxml2.cmake" in CMAKE_MODULE_PATH this project has
    asked CMake to find a package configuration file provided by "tinyxml2",
    but CMake did not find one.
  
    Could not find a package configuration file provided by "tinyxml2" with any
    of the following names:
  
      tinyxml2Config.cmake
      tinyxml2-config.cmake
  
    Add the installation prefix of "tinyxml2" to CMAKE_PREFIX_PATH or set
    "tinyxml2_DIR" to a directory containing one of the above files.  If
    "tinyxml2" provides a separate development package or SDK, be sure it has
    been installed.
  
  
  ---
```
But it works here
https://github.com/ros-controls/ros2_control/blob/master/hardware_interface/CMakeLists.txt#L16
with the capitalized version

```suggestion
  TinyXML2
```